mne_bids_pipeline/_main.py

```diff
@@ -32,46 +35,50 @@ def main():
         filename extension, separated by a '/'. For example, to run ICA, you
         would pass 'sensor/run_ica`. If unspecified, will run all processing
         steps. Can also be a tuple of steps."""))
-    parser.add_option(
+    parser.add_argument(
         '--root-dir', dest='root_dir', default=None,
         help="BIDS root directory of the data to process.")
-    parser.add_option(
+    parser.add_argument(
         '--subject', dest='subject', default=None,
         help="The subject to process.")
-    parser.add_option(
+    parser.add_argument(
         '--session', dest='session', default=None,
         help="The session to process.")
-    parser.add_option(
+    parser.add_argument(
         '--task', dest='task', default=None,
         help="The task to process.")
-    parser.add_option(
+    parser.add_argument(
         '--run', dest='run', default=None,
         help="The run to process.")
-    parser.add_option(
-        '--n_jobs', dest='n_jobs', type='int', default=None,
+    parser.add_argument(
+        '--n_jobs', dest='n_jobs', type=int, default=None,
         help="The number of parallel processes to execute.")
-    parser.add_option(
+    parser.add_argument(
         '--interactive', dest='interactive', action='store_true',
         help="Enable interactive mode.")
-    parser.add_option(
+    parser.add_argument(
         '--debug', dest='debug', action='store_true',
         help="Enable debugging on error.")
-    parser.add_option(
+    parser.add_argument(
         '--no-cache', dest='no_cache', action='store_true',
         help='Disable caching of intermediate results.')
-    options, args = parser.parse_args()
+    options = parser.parse_args()
     config = options.config
+    config_switch = options.config_switch
     bad_msg = (
         'You must specify a configuration file as a single argument '
         'or with --config.'
     )
+    bad = False
     if config is None:
-        if len(args) == 1:
-            config = args[0]
+        if config_switch is None:
+            bad = True
         else:
-            raise ValueError(bad_msg)
-    elif len(args):
-        raise ValueError(bad_msg)
+            config = config_switch
+    elif config_switch is not None:
+        bad = True
+    if bad:
+        parser.error(bad_msg)
```

Comment:
@hoechenberger I replaced the `try/except` stuff with this, I think it's the standard way to handle errors:
```
$ mne_bids_pipeline
usage: mne_bids_pipeline [-h] [--version] [--config FILE] [--steps STEPS] [--root-dir ROOT_DIR] [--subject SUBJECT] [--session SESSION] [--task TASK] [--run RUN]
                         [--n_jobs N_JOBS] [--interactive] [--debug] [--no-cache]
                         [config]
mne_bids_pipeline: error: You must specify a configuration file as a single argument or with --config.
```

Reply:
> @hoechenberger I replaced the `try/except` stuff with this, I think it's the standard way to handle errors:

I saw this too when reading the docs, but I thought using our custom logger would be more consistent … but I'm not sure! Could we maybe insert an emoji here, just to get a little bit more colorful output? :)

Reply:
Done
```
mne_bids_pipeline: error: 😵 You must specify a configuration file either as a single argument or with --config, but neither was provided.
```